src/code/engine_library/runPopulationWorkflow.m

```diff
@@ -79,8 +76,18 @@ function runPopulationWorkflow(TaskList,Settings,PopRunSet,varargin)
     % directory with infos like application protocols, and factor for
     % unitconversion 
     for iSet = 1:length(PopRunSet)
-        successInputCheck = preparePopulationSimulationConsistency(Settings,PopRunSet(iSet)) & successInputCheck;
+        [success,VPC] = preparePopulationSimulation(WSettings,PopRunSet(iSet),VPC);
+        successInputCheck = success & successInputCheck;
     end
```

Comment:
Same as above: ALL simulations are processed even if it fails for one simulation. On purpose?